extension/extension/writing.js

```diff
@@ -734,3 +738,9 @@ WO_XHR.send = function () {\n\
 "
 
 window.addEventListener("load", writing_onload);
+
+window.addEventListener("visibilitychange", () => {
```

Comment:
Give a biref comment here about the purpose.  Because this is open soruce and we expect this to have a long life cycle and be amended it would help.

Reply:
I've added this